redisinsight/api/provider-interfaces/models/command-execution.ts

```diff
@@ -0,0 +1,20 @@
+// todo: discuss. introduce userId here?
+export class CommandExecution {
```

Comment:
@arthosofteq - I'm adding lots of pseudo-documentation questions below. I suggest you add comments to the interface describing each property's purpose and supported values.